model/project_ref.go

```diff
@@ -957,15 +958,20 @@ func (p *ProjectRef) GetProjectSetupCommands(opts apimodels.WorkstationSetupComm
 		if obj.Directory != "" {
 			dir = filepath.Join(dir, obj.Directory)
 		}
+		if err := os.MkdirAll(dir, 0755); err != nil {
+			return nil, errors.Wrapf(err, "can't make directory '%s'", dir)
+		}
 
-		commandNumber := idx + 1 // to avoid logging a stale number
-		cmd := jasper.NewCommand().Directory(dir).AppendArgs("mkdir", "-p", dir).
-			SetErrorSender(level.Error, opts.Output).
+		// avoid logging the final value of obj
+		commandNumber := idx + 1
+		cmdString := obj.Command
+
+		cmd := jasper.NewCommand().SetErrorSender(level.Error, opts.Output).
 			Append(obj.Command).
```

Comment:
can we use cmdString here now as well?

Reply:
we could, but they should both be the same